function main():
	try:
		raise $exception()
	except:
		return 'ok'
	return 'a'
+++value 'ok'
